api/src/main/java/org/openmrs/module/idgen/AutoGenerationOption.java

```diff
@@ -165,4 +165,8 @@ public boolean isAutomaticGenerationEnabled() {
 	public void setAutomaticGenerationEnabled(boolean automaticGenerationEnabled) {
 		this.automaticGenerationEnabled = automaticGenerationEnabled;
 	}
+	
+	public String getUuid() {
+		return this.getIdentifierType().getUuid();
```

Comment:
This is wrong. getUuid should return the class. Since this one has none, you can just return the string version of its id